docker-compose.example-cms.yml

```
@@ -14,7 +14,7 @@ services:
       - core_cms_net
 
   postgres:
-    image: postgres:11.5
+    image: postgres:12
```

This file, besides the CMS image name, is otherwise identical to `docker-compose.dev.yml`. Until I remember out why it exists (J.G. added it), I'd like to keep it in sync.

```suggestion
    image: postgres:14.9
```